server/services/websockets.ts

```diff
@@ -72,20 +75,13 @@ export default function init(
     Metrics.gaugePerInstance("websockets.count", 0);
   });
 
-  // Forbid all unauthenticated connections
```

Comment:
Removed as this was relying on internal properties of SocketIO, it's not actually required in our implementation because the user is only ever subscribed to rooms after authentication is complete. Data is not sent over any default namespaces.